codingTest.js

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
+let input = fs.readFileSync(filePath).toString().split("\n");
+
+function solution() {
+  // 풀이
+  const scores = input[0].split(", ");
+
+  const total = scores.reduce((acc, el) => {
```

Comment:
total은 000해서 옳지못한 변수입니다.
sum으로 바꾸면 000가 좋습니다.

```suggestion
  const sum = scores.reduce((acc, el) => {
```